app/dashboard/subDashboards/controller.js

```diff
@@ -4,17 +4,20 @@ import { createDashboardPageContext } from '../createDashboardPageContext';
 import logger from '../../logger';
 import { apiHost } from '../../config';
 
-export const getSubDashboardPageContext = async (solutionId, sectionId) => {
-  const dashboardManifest = new ManifestProvider().getSubDashboardManifest(sectionId);
+export const getSubDashboardPageContext = async (solutionId, dashboardId) => {
+  const dashboardManifest = new ManifestProvider().getSubDashboardManifest(dashboardId);
 
-  const endpoint = `${apiHost}/api/v1/Solutions/${solutionId}/sections/${sectionId}`;
+  const endpoint = `${apiHost}/api/v1/Solutions/${solutionId}/sections/${dashboardId}`;
```

Comment:
Why are we using dashboardId here? I thought dashboardId was the id of the parent section e.g. browser-based?

Reply:
I just changed it to make it bit more obvious here that we are dealing with dashboard here. It might be worth getting the API to change this endpoint to do likewise

Reply:
is dashboardId the new sectionId or is it the id of parent?

Reply:
We were using the dashboardId and sectionId interchangeably to mean the same thing. But it's starting to get bit confusing now. I've tried to make it bit more clear here that what we are after here is the dashboard (in particular the subDashboard). I think it'll probably make bit more sense when the API endpoint is changed to reflect this also.